src/components/JobConfiguration.vue

```diff
@@ -181,10 +182,13 @@ export default defineComponent({
             role: 'cancel'
           }, {
             text: this.$t('Skip'),
-            handler: () => {
+            handler: async () => {
               if (job) {
-                this.store.dispatch('job/skipJob', job).then(() => {
-                  showToast(translate("This job has been skipped"))
+                await this.store.dispatch('job/skipJob', job).then((resp) => {
```

Comment:
If we are using await, we shouldn't use then

Reply:
Sir, Updated code.